project1/server/file_handler/my_file_handler.h

```diff
@@ -0,0 +1,50 @@
+//
+// Created by jake on 1/25/21.
+//
+
+#ifndef PROJECT1_MY_FILE_HANDLER_H
+#define PROJECT1_MY_FILE_HANDLER_H
+
+#include "file_handler.h"
+#include <iostream>
+#include <unistd.h>
+#include <fstream>
+#include <sys/stat.h>
+
+
+
+
+
+/**
+ * @brief This class represents common file handling operations.
+ *
+ */
+namespace server::file_handler{
+class MyFileHandler: public IFileHandler {
+
+ private:
+  std::ofstream os;
+
+ public:
+
+  std::vector<uint8_t> Get(const std::string& filename) const;
```

Comment:
Nitpick: Specify [`final`](https://en.cppreference.com/w/cpp/language/final) for these declarations.